docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -75,9 +75,9 @@
    "source": [
     "## Protocols\n",
     "\n",
-    "Inheritence is a powerful way to organize code in programming languages that allows you to share code between multiple components of the program.\n",
+    "Inheritance is a powerful way to organize code in programming languages that allows you to share code between multiple components of the program.\n",
     "\n",
-    "In Swift, there are different ways to express inheritence. One of those ways you may be familiar with already - class inheritence. However, Swift has another way - protocols.\n",
+    "In Swift, there are different ways to express inheritance. One of those ways you may be familiar with already - class inheritance. However, Swift has another way - protocols.\n",
```

Comment:
Some unrelated suggestions:
```suggestion
    "In Swift, there are different ways to express inheritance. You may be already be familiar with one of those ways, from other languages: class inheritance. However, Swift has another way: protocols.\n",
```

Reply:
I think you may have an extra "be" in the suggestion: "You may already be familiar..."

Reply:
Thanks for noticing! Fixed the suggestion.